MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+module(
+    name = "highs",
+    version = "0.0.0",
```

Comment:
should version be 1.7.0 (or updated later once everything seems to pass in the bazel central repository side ?)

Reply:
I'm more concerned with a user fetching the repo and having 0.0.0 as version while it is (at least) 1.7.0